hash/mac_flop_orig.xml

```diff
@@ -0,0 +1,437 @@
+				<rom name="star trek- the kobayashi alternative.moof" size="649065" crc="b0a20dad" sha1="1118fa143ccfffc98f01c46e712c81594c9da048" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="macattk">
+		<description>Mac Attack</description>
+		<year>1984</year>
+		<publisher>Miles Computing</publisher>
+		<!-- Compatibility: Mac 128K, Mac 512K, Mac 512Ke, Mac Plus -->
+		<!--"Mac Attack" is a 1984 action game developed by Tim Hays and distributed by Miles Computing. It self-boots and runs on Mac 128K, Mac 512K, Mac 512Ke, and Mac Plus.-->
+
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="648972">
+				<rom name="mac attack.moof" size="648972" crc="d1766425" sha1="4cdcc8508317bae4d91e907b7c42b0e76765bd8b" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="gatov13">
+		<description>GATO (Version 1.3)</description>
+		<year>1985</year>
+		<publisher>Spectrum Holobyte</publisher>
+		<!-- Compatibility: Mac 128K, Mac 512K, Mac 512Ke, Mac Plus, Mac SE, Mac SE FDHD, Mac Classic -->
+		<!--"GATO" is a 1985 simulation game developed by Bill Scott, James Rhodes, Sean Hill, and Mark Dunn, ported to the Macintosh by MindBender, and distributed by Spectrum Holobyte. This is version 1.3. It self-boots and runs on Mac 128K, Mac 512K, Mac 512Ke, Mac Plus, Mac SE, Mac SE FDHD, and Mac Classic.-->
+
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="641368">
+				<rom name="gato v1.3.moof" size="641368" crc="5fffebb8" sha1="fb57fa04198cc1c7d54773a1b9391c782cdaa7bd" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="drkcas10">
+		<description>Dark Castle (Version 1.0)</description>
+		<year>1986</year>
+		<publisher>Silicon Beach Software</publisher>
+		<!-- Compatibility: Mac 512K, Mac 512Ke, Mac Plus -->
+		<!--"Dark Castle" is a 1986 action game developed by Jonathan Gay (programming), Mark Stephen Pierce (design and graphics), Eric Zocher (Real Sound), and Dick Noel (voice characterizations), and distributed by Silicon Beach Software. This is version 1.0. It self-boots and runs on Mac 512K, Mac 512Ke, and Mac Plus.-->
+
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1"/>
+			<dataarea name="flop" size="649028">
+				<rom name="dark castle v1.0 - disk 1.moof" size="649028" crc="05b645ca" sha1="d563bd2d81985ec5416dc806475d1ab49dfa7d49" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2"/>
+			<dataarea name="flop" size="649028">
+				<rom name="dark castle v1.0 - disk 2.moof" size="649028" crc="6c970e1b" sha1="c6f9c8359c36231c3ad772a41701abeb0a0bfbc8" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="oidsv14">
+		<description>Oids (Version 1.4)</description>
+		<year>1992</year>
+		<publisher>FTL Games</publisher>
+		<info name="usage" value="This is not bootable and will need to be inserted after booting the OS"/>
```

Comment:
It seems that `<sharedfeat name="automount" value="no"/>` would be a more proper way of preventing mounting of images at boot time. However, none of the software lists use it currently, and I haven't tested whether it actually works.

Reply:
Maybe this is something @rb6502 could implement if it's not in? I'd be willing to go back and tag any non-bootable Apple II and Mac disks with that if that's the best solution, but I'm going to let the driver maintainer make the judgement call on what's best.

Reply:
Support is there, but as AJR said we have no idea if it works.

Reply:
I just tried, it will just not mount every item listed in the software node.

Reply:
But what if you’re using a Mac with multiple drives and trying to mount multiple software list items, like `-flop1 mac_flop_misc:sys605:flop1 -flop2 mac_flop_orig:oidsv14:flop1`?  Wouldn’t disabling automatic mounting of media just confuse the user?

And while I think of it, a guy on reddit was confused by the way the drive names and part names are both `flop` with a number.  Would it make more sense to use `disk` rather than `flop` in part names to make it more apparent that they aren’t the same thing?